starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,55 @@
 // Names and Input
+console.log("I'm Ready!");
+
+var hacker1 = "Fareaha".split('').join(' ');
+hacker1 = hacker1.toUpperCase();
+console.log("The driver's name is " + hacker1);
+
+var hacker2 = prompt("What is your name?");
+var hacker2_1 = hacker2.split('').reverse().join('');
+console.log("The navigator's name is " + hacker2_1);
 
 
 //Conditionals
+if (hacker1.length > hacker2.length){
+    console.log("The Driver has the longest name, it has " + hacker1.length + " characters");
+  } else if (hacker1 < hacker2.length){
+    console.log("Yo, navigator got the longest name, it has " + hacker2.length + " characters");
+  } else {
+    console.log("wow, you both got equally long names," + hacker1.length + " chsaracters!!");
+  }
+  
+  if (hacker1[0] < hacker2[0]){
+    console.log("The driver's name goes first")
+  }else if (hacker2[0] > hacker1[0]){
+    console.log("Yo, the navigator goes first definitely")
+  } else {
+    console.log("What?! You both got the same name?")
+  }
+
+  //palindrom
+var palindrome = prompt("Is it a palindrome?");
+var palindromeReverse = palindrome.split("").reverse().join("");
+
+if (palindrom = palindromeReverse) {
```

Comment:
2 small typos here : you called your variable  "palindrome" instead of "palindrom" on line 31, and what you wanted to use is the equality operator ("===") instead of the assignment operator ("=") :)